README.md

```diff
@@ -25,14 +25,6 @@ OPTIONAL SECTIONS:
 After you're finished please remove all the comments and instructions!
 -->
 
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
 
 <!-- TABLE OF CONTENTS -->
 
```

Comment:
![Screenshot from 2023-02-16 01-03-36](https://user-images.githubusercontent.com/83042742/219209447-c014cb23-38b0-49c0-a9e2-9ed693a0f2e4.png)
- [ **OPTIONAL** ] As your Hamburger button is already wrapped in a `nav` tag, I think it would be better if you update the above code by wrapping the Hamburger itself  in a `div` tag for semantics best preactices. i.e.:
```HTML
<div class="hamburger">
  <img src="statics/images/Union.png" alt="Menu">
</div> 
```

Reply:
ok,it is done!

Reply:
done